src/vs/workbench/contrib/themes/browser/themes.contribution.ts

```diff
@@ -602,6 +622,15 @@ MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, {
 	order: 3
 });
 
+MenuRegistry.appendMenuItem(MenuId.MenubarPreferencesMenu, {
```

Comment:
The menu bar is only for very few items. Thi command would just be in the command palette, like most commands.

Reply:
Ok, I'll remove it :)